leakcanary-android/src/main/java/com/squareup/leakcanary/internal/LeakCanaryInternals.java

```diff
@@ -50,6 +52,8 @@
 
   private static final Executor fileIoExecutor = newSingleThreadExecutor("File-IO");
 
+  private static final String NOTIFICATION_CHANNEL_ID = "leakcanary";
```

Comment:
scoped to the app, yes.
i was thinking it might be worth doing something like `com.squareup.leakcanary.internal.NOTIFICATION_CHANNEL_ID` to avoid collisions within the app.
unlikely somebody would have a notification channel called "leakcanary" in an app, though.